dcat/examples/csiro-dap-examples.ttl

```diff
@@ -232,13 +232,13 @@ dap:atnf-P366-2003SEPT_1
       rdf:type adms:Identifier ;
       dct:creator <https://www.doi.org/> ;
       skos:notation "10.4225/08/598dc08d07bb7" ;
-      adms:schemeAgency "International DOI Foundation" ;
+      adms:schemeAgency "International DOI Foundation"@en ;

```

Comment:
This is a proper name so I don't think it should have a language-tag

Reply:
```suggestion
      adms:schemeAgency "International DOI Foundation";
```